### DISCUSSION TITLE
Unify dialogs and confirms in Mística Web

### DISCUSSION THREAD

**Discussion by PatriRoman** [Telefonica/mistica-design#1867] [Proposals]
### Goal

Unify visually and behaviourally the dialogs and confirms of Mística Web as nowadays they have a different visual and behaviour. 

### Context

Nowadays in Mística Web, dialogs and confirms have a different visual and behaviour when they really should be unified and have a ‘transparent’ experience with the final user. The difference on web between a confirm and a dialog is simply technical and the user doesn't have to be aware of it. For them, it is simply a lightbox that pops up with information and one or more actions. 
Therefore we propose a visual unification, matching the title of both components and also adding the close cross to the confirms. It can be confusing throughout an experience that two elements that are really one for the final user, some can be closed and others not. 
<img width="571" alt="image" src="https://github.com/user-attachments/assets/ceee803f-ca81-4031-a5cc-e593ef19fcf0">

### Impact

It would have an impact on the visual and behavioural changes of the confirm.

### Discussion Questions

- With all the above said... Does it make sense that if we merge the two of them, one of the components would cease to exist and merge both into one?

**Comment by andredelramo-hm** [comment]
Buenas, @PatriRoman 🙂

Lo hemos estado revisando y diría que tiene sentido que sean componentes diferentes, porque el Confirm se necesita de cara a nativo, pero vemos lógico que tengan los mismo estilos. Actualizaremos los estilos de los títulos y añadiremos una opción más flexible para el cierre en Dialog, ya que es un componente que nos ofrece mayor personalización.
